### ISSUE TITLE
Qt theme?

### ISSUE THREAD

**Issue by Shadowblitz16** [luigifab/human-theme#12]

**Comment by luigifab** [comment]
No, but "qt5 gtk**2** platformtheme" apply the GTK-2 theme into QT.
I have not searched how to create a QT theme.

**Comment by luigifab** [comment]
I found an "easy" way to do that.
Come back next year for a new life!

<img width="648" height="324" alt="Image" src="https://github.com/user-attachments/assets/20c4ce8f-6824-4eed-b27b-5959fd37745f" />

**Comment by luigifab** [comment]
Alright. It's now done.
